packages/dashboard/src/components/DashboardGridLayout/examples/DGLWithAddCardPanel.Example.tsx

```diff
@@ -0,0 +1,285 @@
+      addCardPanelHeader: 'Billing',
+      addCardPanelImageUrl: '../../../../public/images/DetailsList.svg'
+    }
+  },
+  {
+    id: '8',
+    cardFrameContent: cardFrameContent,
+    header: getHeader('8'),
+    cardContentList: contentAreaList,
+    cardSize: CardSize.mediumWide,
+    x: 2,
+    y: 0,
+    addCardInfo: {
+      addCardPanelBodyText: 'Get guidance on deploying Microsoft 365 features and services, and get training for you and your end users.',
+      addCardPanelHeader: 'Training and guides',
+      addCardPanelImageUrl: '../../../../public/images/Donut.svg'
+    }
+  },
+  {
+    id: '9',
+    cardFrameContent: cardFrameContent,
+    header: getHeader('9'),
+    cardContentList: contentAreaList,
+    cardSize: CardSize.mediumTall,
+    x: 0,
+    y: 1,
+    addCardInfo: {
+      addCardPanelBodyText: 'Install or deploy Office 365 ProPlus software and see how many licensed users have activated it.',
+      addCardPanelHeader: 'Office 365 software',
+      addCardPanelImageUrl: '../../../../public/images/LineChart.svg'
+    }
+  },
+  {
+    id: '10',
+    cardFrameContent: cardFrameContent,
+    header: getHeader('10'),
+    cardContentList: contentAreaList,
+    cardSize: CardSize.small,
+    x: 0,
+    y: 2,
+    addCardInfo: {
+      addCardPanelBodyText: "Stay Informed of new products, features, and changes as they're announced.",
+      addCardPanelHeader: 'Message Center',
+      addCardPanelImageUrl: '../../../../public/images/StackedBarChart.svg'
+    }
+  },
+  {
+    id: '11',
+    cardFrameContent: cardFrameContent,
+    header: getHeader('11'),
+    cardContentList: contentAreaList,
+    cardSize: CardSize.mediumWide,
+    x: 0,
+    y: 2,
+    addCardInfo: {
+      addCardPanelBodyText: "Stay Informed of new products, features, and changes as they're announced.",
+      addCardPanelHeader: 'Message Center',
+      addCardPanelImageUrl: '../../../../public/images/ThumbnailList.svg'
+    }
+  }
+];
+
+export interface IDGLWithAddCardPanelState {
+  isOpen: boolean;
+}
+
+export class DGLWithAddCardPanelExample extends React.Component<{}, IDGLWithAddCardPanelState> {
+  constructor(props: {}) {
+    super(props);
+    this.state = {
+      isOpen: false
+    };
+  }
+  public render(): JSX.Element {
+    return (
+      <>
+        <button onClick={this._openAddCardPanel}>Add Card</button>
+        <DashboardGridLayoutWithAddCardPanel
+          cardsVisibleInAddCardPanel={cardsVisibleInAddCardPanel}
+          cardsVisibleInLayout={cardsVisibleInLayout}
+          isOpen={this.state.isOpen}
+          sectionTitle={'First section'}
+          panelHeader={'Drag cards to your home page'}
+          onLayoutChange={this._onLayoutChange}
+        />
+      </>
+    );
+  }
+
+  private _openAddCardPanel = () => {
+    this.setState({ isOpen: true });
+  };
+
+  private _onLayoutChange(newLayout: DashboardGridBreakpointLayouts): void {
+    console.log('The new layout is: ', newLayout);
```

Comment:
>console.log('The new layout is: ', newLayout); [](start = 4, length = 46)

remove

Reply:
I thought this way user can look at the layout returned. Also if we show an example of callback, and if we leave the callback empty tslint will complain. But if you want me to just leave the callback empty will leave a //tslint: disable comment